### ISSUE TITLE
[🐛 Bug]: GA doesn't load on system startup

### ISSUE THREAD

**Issue by cmt29** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#588]
### What went wrong?

On system startup and with `Launch at System Startup` toggled on, GA doesn't appear in system tray, nor respond to hot word or SUPER+SHIFT+A, indicating that it hasn't been loaded in background either.

### What did you expect to happen?

GA to appear in system tray, or to respond to hot word or SUPER+SHIFT+A and appear following system startup.

### Steps to reproduce the bug

1. Startup or Reboot System
2. Press SUPER+SHIFT+A or speak hot word

### App Version

v1.0.0

### OS Version

OpenSuse Leap 15.2 (Kernel 5.3.18-lp152.106)

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by im-coder-lg** [comment]
Can you really confirm via `ps all` in the terminal? That might seem unrelated, but if there are things that don't have the word `g-assist` or anything similar, it really doesn't start up on boot which is a bug. Can you confirm if it works when you add it to the app starter at boot app? I think there's something like that so check any app that executed commands during system startup, and add GAUDC there.

**Comment by cmt29** [comment]
Yes, I can confirm that `ps all` shows nothing with `g-assist` or similar.
I can add it manually using KDE system settings and it starts up fine. But
simply toggling the option in the GAUDC settings doesn't seem to work.

**Comment by im-coder-lg** [comment]
Hmmm... New bug, till then use the KDE settings until this is fixed. Once it's fixed this issue will close automatically.
